src/app/manage-projects/manage-single-project/manage-single-project.component.ts

```diff
@@ -52,10 +52,7 @@ export class ManageSingleProjectComponent implements OnInit, AfterViewInit {
 
     fetchAllData() {
         this.getProject(() => {
-            SpinnerUtil.showSpinner();
```

Comment:
I didn't mean that spinners should be removed from `fetchAllData()`, but should be set properly: 
1. on the entrance of the function
2. in every possible callback where it should be closed (all callbacks that finishes data fetch; success and error callbacks).